### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by fbartels**
@erik-wramner some smaller updates following the switch to the Dovecot repo. Still want to look into the Dockerfile and the new hadolint exception later this day.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -132,6 +130,10 @@ RUN sed -i -e 's/include_try \/usr\/share\/dovecot\/protocols\.d/include_try \/e
   sed -i -e 's/^.*lda_mailbox_autosubscribe.*/lda_mailbox_autosubscribe = yes/g' /etc/dovecot/conf.d/15-lda.conf && \
   sed -i -e 's/^.*postmaster_address.*/postmaster_address = '${POSTMASTER_ADDRESS:="postmaster@domain.com"}'/g' /etc/dovecot/conf.d/15-lda.conf && \
   sed -i 's/#imap_idle_notify_interval = 2 mins/imap_idle_notify_interval = 29 mins/' /etc/dovecot/conf.d/20-imap.conf && \
+  # Adapt mkcert for Dovecot community repo
+  sed -i 's/CERTDIR=.*/CERTDIR=\/etc\/dovecot\/ssl/g' /usr/share/dovecot/mkcert.sh && \
+  sed -i 's/KEYDIR=.*/KEYDIR=\/etc\/dovecot\/ssl/g' /usr/share/dovecot/mkcert.sh && \
+  sed -i 's/KEYFILE=.*/KEYFILE=\$KEYDIR\/dovecot.key/g' /usr/share/dovecot/mkcert.sh && \
   # stretch-backport of dovecot needs this folder
```
Comment (erik-wramner) [inline Dockerfile]: Remove this comment as well or reword it. Keeping folder for backwards compatibility with stretch-backports for example.

File: README.md [hunk 2]
```diff
@@ -9,24 +9,25 @@ Easy to deploy and upgrade.
 
 Includes:
 
-- postfix with smtp or ldap auth
-- dovecot for sasl, imap (and optional pop3) with ssl support, with ldap auth
+- [Postfix](http://www.postfix.org) with smtp or ldap auth
+- [Dovecot](https://www.dovecot.org) for sasl, imap (and optional pop3) with ssl support, with ldap auth
+  - Divecot ist installed from the [Dovecot Community Repo](https://wiki2.dovecot.org/PrebuiltBinaries)
```
Comment (erik-wramner) [inline README.md]: FIx spelling, Divecot should be Dovecot

**Comment by fbartels** [comment]
Yes indeed. The sc2016 was because of the quoting in the sed commands. I don't really see a way around how its currently done, though, therefore I opted to ignore it in the past.

**Comment by fbartels** [comment]
Fixed that last typo and squashed down commits. Before the squash the tests were green, so that should still be the case now as well.

**Pull request merged**